examples/policygen/generated/forseti_policies/policies/templates/gcp_gke_enable_binauthz_v1.yaml

```diff
@@ -0,0 +1,69 @@
+# Copyright 2019 Google LLC
```

Comment:
2021

Reply:
These files are owned by another team in repo: https://github.com/GoogleCloudPlatform/policy-library. We are just pulling those files from there. I just rerun regen to get the latest files so the presubmit tests for other PRs won't fail.

@thomasfung-hk is working on either pin the version of that repo in our repo to a fixed commit so it will not always try to pull from the HEAD or just remove the policy generator part from our presubmit.